components/Base/CollapsableItem.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <Disclosure v-slot="{ open }">
+    <DisclosureButton
+      class="h-14 w-full border-b border-solid border-b-gray-200 text-sm font-normal leading-normal flex justify-between items-center"
+    >
+      {{ info.title }}
+      <PlusIcon
+        v-show="!open"
+        class="inline-block"
+      />
+      <MinusIcon
+        v-show="open"
+        class="inline-block"
+      />
+    </DisclosureButton>
+    <DisclosurePanel class="text-xs mt-3 mb-3">
+      {{ info.description }}
+    </DisclosurePanel>
+  </Disclosure>
+</template>
+
+<script setup lang="ts">
+import { Disclosure, DisclosureButton, DisclosurePanel } from '@headlessui/vue';
+import PlusIcon from '~/assets/icons/plus.svg?skipsvgo';
+import MinusIcon from '~/assets/icons/minus.svg?skipsvgo';
```

Comment:
I think it would be great to add some wrapper component for all icons, but now we can do it this wat. 

@WawrzenskiPatryk What do you think about to create component like JIcon in Jaltech?

Reply:
There is [vite-svg-loader](https://www.npmjs.com/package/vite-svg-loader) in use. It wraps icons into components for us 😄 

```vue
<template>
  <MyIcon />
</template>

<script setup>
import MyIcon from './my-icon.svg'
</script>
```

Reply:
Ah yea, got it 👍

Reply:
So in general I think there is no need to add any additional handling for Icons, we import svgs now just as we would import components 😄

Reply:
This is really cool if it works fine, so everything is clear right now 😄

Reply:
is this not handles by `svgLoader` already?

Reply:
No it handles, so no changes need here